requirements.txt

```diff
@@ -0,0 +1,5 @@
+mkdocs
```

Comment:
Please use pinned version

```
mkdocs==1.1.2
mkdocs-material==6.1.6
mkdocs-same-dir==0.1.0
mkdocs-literate-nav==0.2.0
mkdocs-section-index==0.2.0
```

Reply:
Hmm OK. It's just that there's no easy mechanism to auto upgrade versions in such a file.
Would you agree with `~=1.1.2`? (`>=1.1.2, ==1.1.*`)

Reply:
Oh I found a good way. Done.